### DISCUSSION TITLE
Redirection url on the insight counter.

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#11862] [Feature Requests]
Redirection url on the insight counter. It is also helpful if we can embed the filters in URL so relevant data will be displayed automatically
![unknown (2)](https://user-images.githubusercontent.com/36895131/155760305-d4c0830d-8f16-42a1-bda9-d332b48b7f32.png)

**Comment by rijkvanzanten** [comment]
Try using the "Button Links" presentation interface 🙂 

<img width="776" alt="CleanShot 2022-02-28 at 11 39 28@2x" src="https://user-images.githubusercontent.com/9141017/156022131-5087f97b-d4a2-4ce7-b625-25f3b405040e.png">

<img width="987" alt="CleanShot 2022-02-28 at 11 39 45@2x" src="https://user-images.githubusercontent.com/9141017/156022159-fdc45924-7688-44f0-922c-50f451e776ab.png">

**Reply by Jeyakumarrathnaselvi** [reply]
No, we don't button link option in the insights sections

![image](https://user-images.githubusercontent.com/36895131/159130304-6306303f-22be-46c9-af35-c389be261efc.png)

**Reply by rijkvanzanten** [reply]
Ahh gotcha. I thought you were talking about regular fields 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
